03-csv-vector-raster-plotting.Rmd

```diff
@@ -325,7 +329,7 @@ plot(extent(aoiBoundary_HARV),
      lwd=6,
      col="springgreen")
 
-legend("left",
+legend("bottom",
```

Comment:
Yup. I selected the left because you could read the text (it was in the white space).

Let's leave it as an issue but keep it on the left for the time being. Let someone else fix it. It's really minor.